fuzz/fuzz.js

```diff
@@ -89,7 +99,7 @@ const output = [
   separator,
   JSON.stringify(options, null, 2),
   separator,
-  message
+  message + ' after ' + tryCount + ' tries',
```

Comment:
Helps give a sense of how much it had to work to find a broken example

![image](https://cloud.githubusercontent.com/assets/197597/22179452/1863ae2c-e009-11e6-987d-c444c164ba06.png)